proto/observer/tx.proto

```diff
@@ -113,3 +114,10 @@ message MsgVoteTSSResponse {
   bool vote_finalized = 2;
   bool keygen_success = 3;
 }
+
+message MsgUpdateParams {
```

Comment:
Only `BallotMaturityBlocks` is still used in `observer` which can be moved to `emissions`

What about already moving it in the emissions module in this PR?